### PR TITLE
shortcodes for compliance phase chef generators

### PR DESCRIPTION
**Pull request by vkarve-chef**
Signed-off-by: Vikram Karve <vikram.karve@progress.com>

## Description

Added shortcodes and menu entries for the compliance phase generators introduced in chef-cli.

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Comment by IanMadd** [comment]
Where are these used?

**Comment by vkarve-chef** [comment]
@IanMadd these will be used in chef-workstation docs alongside other `chef generate ...` commands.

File: themes/docs-new/layouts/shortcodes/ctl_chef_generate_waiver.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Use the `chef generate waiver` subcommand to generate a [waiver](https://docs.chef.io/inspec/waivers/) file for the Chef Infra Client's Compliance Phase execution.
```
Comment (IanMadd) [inline themes/docs-new/layouts/shortcodes/ctl_chef_generate_waiver.md]:
```suggestion
Use the `chef generate waiver` subcommand to generate a [waiver](/inspec/waivers/) file for the Chef Infra Client's Compliance Phase execution.
```

File: themes/docs-new/layouts/shortcodes/ctl_chef_generate_profile.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Use the `chef generate profile` subcommand to generate a [profile](https://docs.chef.io/inspec/profiles/) for the Chef Infra Client's Compliance Phase execution.
```
Comment (IanMadd) [inline themes/docs-new/layouts/shortcodes/ctl_chef_generate_profile.md]:
```suggestion
Use the `chef generate profile` subcommand to generate a [profile](/inspec/profiles/) for the Chef Infra Client's Compliance Phase execution.
```

File: themes/docs-new/layouts/shortcodes/ctl_chef_generate_input.md [hunk 3]
```diff
@@ -0,0 +1 @@
+Use the `chef generate input` subcommand to generate an [input](https://docs.chef.io/inspec/inputs/) file for the Chef Infra Client's Compliance Phase execution.
```
Comment (IanMadd) [inline themes/docs-new/layouts/shortcodes/ctl_chef_generate_input.md]:
```suggestion
Use the `chef generate input` subcommand to generate an [input](/inspec/inputs/) file for the Chef Infra Client's Compliance Phase execution.
```

**Comment by IanMadd** [comment]
Can you remove the last commit ([9e8c04c](https://github.com/chef/chef-web-docs/pull/3713/commits/9e8c04c43101d97f50fb4f0abad0777b8530e8f5)) and then we'll redo the menu nav changes with #3731 after we update the content in chef/chef-workstation.

**Review by IanMadd — APPROVED** [review]
LGTM

**Pull request merged**